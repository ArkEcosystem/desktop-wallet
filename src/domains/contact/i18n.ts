export const translations: { [key: string]: any } = {
	CONTACTS_PAGE: {
		ADD_CONTACT: "Add Contact",
		ADD_CONTACT_MESSAGE: "Add your most frequent contacts for fast, easy payments",
		TITLE: "My Contacts",
		SUBTITLE: "Manage your frequent contacts",
	},
	CONTACT_FORM: {
		ADD_ADDRESS: "Add address",
		ADDRESS: "Address",
		ADDRESSES: "Addresses",
		DELETE_CONTACT: "Delete Contact",
		NAME: "Name",
		NETWORK: "Network",
	},
	MODAL_CREATE_CONTACT: {
		TITLE: "Add Contact",
		DESCRIPTION: "Enter your contacts name and addresses and add them to your list.",
	},
	MODAL_DELETE_CONTACT: {
		TITLE: "Delete Contact",
		DESCRIPTION: "Do you really want to delete this contact? Once deleted, you will not be able to restore it.",
	},
	MODAL_UPDATE_CONTACT: {
		TITLE: "Edit Contact",
	},
	MODAL_SEARCH_CONTACT: {
		TITLE: "Search Contact",
		DESCRIPTION: "Find the contact you need quickly and conveniently",
		PLACEHOLDER: "Search...",
	},
	VALIDATION: {
		MAXLENGTH_ERROR: "The Contact Name should have less than {{maxLength}} characters.",
		CONTACT_NAME_EXISTS: "Contact with name {{name}} already exists.",
		CONTACT_NAME_EXISTS_SUFFIX: "already exists",
	},
};
