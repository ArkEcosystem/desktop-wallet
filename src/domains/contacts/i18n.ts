export const translations: { [key: string]: any } = {
	CONTACT_FORM: {
		ADD_ADDRESS: "Add Address",
		ADDRESS: "Address",
		ADDRESSES: "Addresses",
		NAME: "Name",
		NETWORK: "Network",
	},
	MODAL_CREATE_CONTACT: {
		TITLE: "Add Contact",
		DESCRIPTION: "Enter your contacts name and addresses and add them to your list.",
	},
	MODAL_DELETE_CONTACT: {
		TITLE: "Delete Contact",
		DESCRIPTION: "Do you really want to delete this contact? Once deleted, you will not be able to restore it.",
	},
	MODAL_UPDATE_CONTACT: {
		TITLE: "Edit Contact",
	},
};
