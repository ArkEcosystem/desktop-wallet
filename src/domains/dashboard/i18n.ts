export const translations: { [key: string]: any } = {
	FILTER_WALLETS: {
		NETWORK: {
			TITLE: "Filter Network",
			DESCRIPTION: "Select the types of networks",
		},
		WALLETS: {
			TITLE: "Your Wallets",
			DESCRIPTION: "Select the display of your wallets",
		},
		CHART: {
			TITLE: "Portfolio Chart",
			DESCRIPTION: "You can hide this unit if you like",
		},
		TRANSACTION_HISTORY: {
			TITLE: "Transaction History",
			DESCRIPTION: "You can hide this unit if you like",
		},
	},

	TRANSACTION_HISTORY: {
		TITLE: "Transactions History",
		EMPTY_TEXT:
			"This will display the history of your transactions. But you don't have more than one transaction at the moment.",
	},

	WALLET_CONTROLS: {
		CREATE: "Create",
		IMPORT: "Import",
	},

	DASHBOARD_PAGE: {
		CHART: {
			TITLE: "Portfolio Chart",
			PERCENTAGES_LABEL: "Total portfolio",
		},
	},
};
