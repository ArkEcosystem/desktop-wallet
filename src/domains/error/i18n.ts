export const translations: { [key: string]: any } = {
	APPLICATION: {
		TITLE: "Something went wrong",
		DESCRIPTION: "It looks like something went wrong. Try visiting another page or reloading.",
		RELOAD: "Reload",
	},
	OFFLINE: {
		TITLE: "Cannot Connect",
		DESCRIPTION: "Check your internet connection if it doesn't work, check it again.",
	},
};
