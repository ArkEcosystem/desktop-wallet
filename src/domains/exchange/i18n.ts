export const translations: { [key: string]: any } = {
	MODAL_ADD_EXCHANGE: {
		TITLE: "Exchange",
		DESCRIPTION:
			"Buy ARK directly from the ARK Desktop Wallet or through one of 20+ exchanges supplying ARK tokens using BTC, ETH or other trading pairs.",
	},
};
