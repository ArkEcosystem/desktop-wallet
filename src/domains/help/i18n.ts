export const translations: { [key: string]: any } = {
	MODAL_CONTACT_US: {
		TITLE: "Contact Us",
		DESCRIPTION: "We'd be happy to answer your questions",
		FIELD_NAME: "Name",
		FIELD_EMAIL: "Email",
		FIELD_SUBJECT: "Subject",
		FIELD_MESSAGE: "Message",
		MESSAGE_REQUIRED: "Message is Required",
		SUBJECT_REQUIRED: "Subject is Required",

		SUBJECT_OPTION: {
			SECURITY: "Security",
			OTHER: "Other",
		},
	},
};
