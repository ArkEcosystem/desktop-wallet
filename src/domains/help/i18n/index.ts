import enUS from "./locales/en-US";

export default {
	"en-US": enUS,
};
