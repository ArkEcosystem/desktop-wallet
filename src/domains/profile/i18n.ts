export const translations: { [key: string]: any } = {
	MODAL_PROFILE_CREATED: {
		TITLE: "Profile Created!",
		DESCRIPTION_1: "If you are new to the ARK Desktop Wallet, view our tutorial to get started.",
		DESCRIPTION_2: "If not, you can skip the tutorial to go directly to your portfolio.",
		START_TUTORIAL: "Start Tutorial",
		SKIP_TUTORIAL: "Skip Tutorial",
	},
};
