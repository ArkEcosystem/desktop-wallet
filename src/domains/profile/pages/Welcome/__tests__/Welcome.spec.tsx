import React from "react";
import { render } from "@testing-library/react";
import { I18nextProvider } from "react-i18next";

import { i18n } from "app/i18n";

import { Welcome } from "../";

describe("Welcome", () => {
	it("should render", () => {
		const profiles = [
			{
				id: 1,
				name: "Oleg Gelo",
				balance: "234,500.46 USD",
				avatar: "https://www.w3schools.com/howto/img_avatar.png",
			},
		];

		const { container, asFragment } = render(
			<I18nextProvider i18n={i18n}>
				<Welcome profiles={profiles} />
			</I18nextProvider>,
		);

		expect(container).toBeTruthy();
		expect(asFragment()).toMatchSnapshot();
	});
});
