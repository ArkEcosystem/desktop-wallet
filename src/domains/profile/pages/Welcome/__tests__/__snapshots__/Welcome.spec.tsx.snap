// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should render 1`] = `
<DocumentFragment>
  <div
    class="w-full h-full"
  >
    <nav
      class="bg-white shadow-header-smooth relative"
    >
      <div
        class="px-4 sm:px-6 lg:px-8"
      >
        <div
          class="relative flex justify-between h-20 md:h-24"
        >
          <div
            class="flex-shrink-0 flex items-center"
          >
            <div
              class="flex rounded-lg p-2 bg-logo"
            >
              <img
                alt="ARK Logo"
                class="h-6 md:h-8 lg:h-10"
                src="ark-logo.png"
              />
            </div>
          </div>
        </div>
      </div>
    </nav>
    <div
      class="flex flex-col items-center justify-center text-center"
    >
      <h1
        class="mb-8 text-4xl font-bold"
      >
        Welcome to ARK
      </h1>
      <div
        class="container w-3/5 mx-auto"
      >
        <img
          alt="Onboarding Banner"
          src="onboarding-banner.svg"
        />
      </div>
      <div
        class="container w-2/5 mx-auto"
      >
        <div
          class="w-full my-8"
        >
          <h1
            class="mx-4 text-2xl font-bold md:mx-8 xl:mx-16"
          >
            Select Profile
          </h1>
          <div
            class="mx-4 mt-2 text-theme-neutral-dark md:mx-8 xl:mx-16"
          >
            You already have a profile, you can choose any of them
          </div>
        </div>
        <div
          class="sc-AxirZ bEjdEN"
        >
          <div
            class="relative px-6 sm:flex sm:items-center py-4"
          >
            <button
              class="absolute top-0 -mt-2 -mr-4 right-0 flex items-center justify-center w-6 h-6 p-1 mt-3"
            >
              <div
                class="sc-AxiKw dklFyP"
                height="20"
                width="20"
              >
                <div>
                  <div>
                    <div
                      data-src="settings.svg"
                      src=""
                    />
                  </div>
                </div>
              </div>
            </button>
            <div
              class="flex flex-row justify-between w-full"
            >
              <div
                class="flex items-center"
              >
                <div
                  class="block w-12 h-12 mx-auto rounded-full sm:mx-0 sm:flex-shrink-0"
                >
                  <img
                    alt="User Avatar"
                    class="rounded-full"
                    data-testid="profile-card__user--avatar"
                    src="https://www.w3schools.com/howto/img_avatar.png"
                  />
                </div>
                <div
                  class="mt-4 text-center sm:mt-0 sm:ml-4 sm:text-left"
                >
                  <p
                    class="text-sm font-semibold text-theme-neutral"
                  >
                    Name
                  </p>
                  <p
                    class="font-semibold text-theme-neutral-dark"
                    data-testid="profile-card__user--name"
                  >
                    Oleg Gelo
                  </p>
                </div>
              </div>
              <div
                class="flex items-center"
              >
                <div
                  class="mt-4 text-center sm:mt-0 sm:ml-4 sm:text-right"
                >
                  <p
                    class="text-sm font-semibold text-theme-neutral"
                  >
                    Total Balance
                  </p>
                  <p
                    class="font-semibold text-theme-neutral-dark"
                    data-testid="profile-card__user--balance"
                  >
                    234,500.46 USD
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="max-w-lg mx-auto mt-10 border-t border-neutral-light"
        />
      </div>
      <div
        class="mx-4 mt-8 mb-4 text-theme-neutral-dark md:mx-8 xl:mx-16"
      >
        Create a new Profile or login with your MarketSquare account to get started
      </div>
      <div
        class="flex justify-center w-full mb-10"
      >
        <button
          class="sc-AxjAm kujxw w-1/5 mr-2"
          color="primary"
        >
          Sign in to MarketSquare
        </button>
        <button
          class="sc-AxjAm lhSfbA w-1/5"
          color="primary"
        >
          Create Profile
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
